### PR TITLE
Microsoft Graph Security Connector

### PR DESCRIPTION
**Pull request by mamurshe**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: connectors/MicrosoftGraphSecurity/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions"
+    ],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "c4829704-0edc-4c3d-a347-7c4a67586f3c",
```
Comment (JatinSanghvi) [inline connectors/MicrosoftGraphSecurity/apiProperties.json]: Was it decided to let everyone view connector's client ID?

File: connectors/MicrosoftGraphSecurity/apidefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1386 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Graph Security",
+    "description": "The Microsoft Graph Security connector helps to connect different Microsoft and partner security products and services, using a unified schema, to streamline security operations, and improve threat protection, detection, and response capabilities. Learn more about integrating with the Microsoft Graph Security API at https://aka.ms/graphsecuritydocs",
+    "version": "1.0",
+    "contact": {
+      "name": "Sips Api team",
+      "email": "sipsisgdev@microsoft.com"
```
Comment (JatinSanghvi) [inline connectors/MicrosoftGraphSecurity/apidefinition.swagger.json]: Similarly, should we let everyone view team's email address?

File: connectors/MicrosoftGraphSecurity/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions"
+    ],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "client_id",
```
Comment (dmkuby) [inline connectors/MicrosoftGraphSecurity/apiProperties.json]: Should it be a parameter? constant? removed?

File: connectors/MicrosoftGraphSecurity/apidefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1386 @@
...
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Graph Security",
+    "description": "The Microsoft Graph Security connector helps to connect different Microsoft and partner security products and services, using a unified schema, to streamline security operations, and improve threat protection, detection, and response capabilities. Learn more about integrating with the Microsoft Graph Security API at https://aka.ms/graphsecuritydocs",
+    "version": "1.0",
+    "contact": {
+      "name": "team_name",
+      "email": "contact_email"
+    },
+    "x-ms-api-annotation": {
```
Comment (dmkuby) [inline connectors/MicrosoftGraphSecurity/apidefinition.swagger.json]: Do we want to expose it in public repo?

File: connectors/MicrosoftGraphSecurity/apidefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,1386 @@
...
+          "400": {
+            "description": "Bad Request"
+          },
+          "403": {
+            "description": "Unauthorized, get admin access grant"
+          },
+          "500": {
+            "description": "Internal Server Error"
+          },
+          "default": {
```
Comment (dmkuby) [inline connectors/MicrosoftGraphSecurity/apidefinition.swagger.json]: Can we remove this from public version? It is not really needed, also '500' and 'default' may bring a confusion and we should discourage authors to specify them in swagger.

**Pull request merged**